### PR TITLE
completed get-articles

### PR DESCRIPTION
**Pull request by Kamz1991**

### CODE REVIEW

File: __tests__/project.test.js [hunk 1]
```diff
@@ -150,3 +150,29 @@ describe("GET /api/articles/article:id/comment count", () => {
     });
   });
 });
+
+describe.only(" GET /api/articles", () => {
```
Comment (dentednerd) [inline __tests__/project.test.js]: Don't forget to remove that .only when you're done with it!

**Review by dentednerd — APPROVED** [review]
Fab stuff Kamahl!

File: __tests__/project.test.js (hunk 1 above)
Comment (Kamz1991) [reply __tests__/project.test.js]: hehe woops

Comment (Kamz1991) [reply __tests__/project.test.js]: sorted it and pushed it up again 
:)

**Pull request merged**